index.html

```diff
@@ -19,6 +19,17 @@
 <link rel="stylesheet" type="text/css" href="css/menu.css" />
 <link rel="stylesheet" type="text/css" href="lib/Font-Awesome/css/font-awesome.min.css" />
 </head>
+
+<script>
+  try {
```

Comment:
Please add some documentation explaining what you are doing and why you are doing it in this file.